spec/Spec/PolynomialsTV.cry

```diff
@@ -0,0 +1,8238 @@
+/**
+ * This module contains test vectors for the `Polynomials` module.
+ */
+module PolynomialsTV where
+
+import Spec::BlsHelpers
+import Spec::Permutations
+import Spec::Polynomials
+
+/**
+ * Verify that the Cryptol spec correctly computes the `roots_of_unity_brp`
+ *  which is used in multiple functions.
+ * @see https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/polynomial-commitments.md#evaluate_polynomial_in_evaluation_form
+ * @see https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/polynomial-commitments.md#compute_quotient_eval_within_domain
+ * @see https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/polynomial-commitments.md#compute_kzg_proof_impl
+ * ```repl
+ * :prove correct_roots_of_unity_brp
+ * ```
+ */
+correct_roots_of_unity_brp : Bit
+property correct_roots_of_unity_brp = roots_of_unity_brp == expected_roots_of_unity_brp where
+    roots_of_unity_brp = bit_reversal_permutation`{len=FIELD_ELEMENTS_PER_BLOB} (compute_roots_of_unity `(FIELD_ELEMENTS_PER_BLOB))
+
+/**
+ * Verify that the Cryptol spec correctly computes the `evaluate_polynomial_in_evaluation_form`.
+ * This test vector was generated from the Deneb Python specification.
+ * @see https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/polynomial-commitments.md#evaluate_polynomial_in_evaluation_form
+ * ```repl
+ * :prove correct_evaluate_polynomial_in_evaluation_form
+ * ```
+ */
+correct_evaluate_polynomial_in_evaluation_form : Bit
+property correct_evaluate_polynomial_in_evaluation_form =
+    evaluate_polynomial_in_evaluation_form eval_poly_polynomial eval_poly_z == expected_eval_poly_output
+
+expected_roots_of_unity_brp = [
```

Comment:
actual suggestion: give this an all-caps name to make it more obvious in the property that it's a constant? I'm not entirely sold on this because I'm not sure it's cryptol-standard, but it does match your spec's standards.

Reply:
Fixed here: 0763ffe84e98ab5c833c2d7b3b4a34317f88d24c